etc/utils/src/file-configs.ts

```diff
@@ -114,3 +121,12 @@ export function getConfigsFolderPath({
 }) {
     return path.join(pathToHome, 'chains', chain, configsFolder ?? 'configs', configsFolderSuffix ?? '');
 }
+
+export function replaceAggregatedBlockExecuteDeadline(pathToHome: string, fileConfig: any, value: number) {
+    const generalConfigPath = getConfigPath({ pathToHome, chain: fileConfig.chain, config: 'general.yaml' });
+    const generalConfig = fs.readFileSync(generalConfigPath, 'utf8');
+    const regex = /aggregated_block_execute_deadline:\s*\d+/g;
+    const newGeneralConfig = generalConfig.replace(regex, `aggregated_block_execute_deadline: ${value}`);
+
+    fs.writeFileSync(generalConfigPath, newGeneralConfig, 'utf8');
+}
```

Comment:
why you are using regex? 
you could properly parse yaml

Reply:
At first I tried to parse the yaml file js' `yaml` package, modify the property and serialize the JS object back to a yaml file, but I encountered some incompatibility:
- for some reason js' yaml package would would convert a `1.0` to an integer when deserializing.
- and would interpret every address `0x...` as an hex value and convert it to an integer

I tried different deserialization oprions ([schema options](https://eemeli.org/yaml/#document-options): core, failsafe, and json), but they all fall short when it's time to deserialize the new yaml file in rust.